.cargo/config.toml

```diff
@@ -1,28 +1,35 @@
-[build]
-rustflags = [
-    "-W", "missing_docs",
-]
-incremental = true
-
-[target.x86_64-apple-darwin]
-rustflags = [
-    "-W", "missing_docs",
-    "-C", "link-arg=-mmacosx-version-min=11.0",
-]
-
-[target.aarch64-apple-darwin]
-rustflags = [
-    "-W", "missing_docs",
-    "-C", "link-arg=-mmacosx-version-min=11.0",
-]
-
-[profile.release]
-strip = true
-
-[env]
-LLVM_SYS_170_PREFIX = { value = "./target-llvm/target-final/", relative = true, force = false }
-
-[tools.clippy]
-warn = [
-    "missing_docs_in_private_items",
-]
+[build]
+rustflags = [
+    "-W", "missing_docs",
+]
+incremental = true
+
+[target.x86_64-apple-darwin]
+rustflags = [
+    "-W", "missing_docs",
+    "-C", "link-arg=-mmacosx-version-min=11.0",
+]
+
+[target.aarch64-apple-darwin]
+rustflags = [
+    "-W", "missing_docs",
+    "-C", "link-arg=-mmacosx-version-min=11.0",
+]
+
+[target.aarch64-unknown-linux-musl]
+linker = "rust-lld"
+
+[target.aarch64-unknown-linux-musl.compiler-rt]
+rustc-link-search = ["./target-llvm/target-host/lib/clang/17/lib/aarch64-unknown-linux-musl"]
```

Comment:
This won't work for local setups where LLVM isn't in `./target-llvm/` :(

Reply:
@hedgar2017 ,

Could we live with that? I believe in 99.99% of cases when `aarch64-unknown-linux-musl` is in use (build and release for this target mostly), `./target-llvm` will always be in place.

The problem here is that using env variables here is not possible, and it looks like the only solution without adding `RUSTFLAGS` and documenting it everywhere.

Reply:
Yeah, it's acceptable.